newdle/client/src/actions.js

```diff
@@ -99,6 +100,13 @@ export function removeParticipant(participant) {
   return {type: REMOVE_PARTICIPANT, participant};
 }
 
+export function fetchParticipant(newdleCode, participantCode, action = PARTICIPANT_RECEIVED) {
```

Comment:
Are there any cases in which we will pass a custom `action` to this function? I don't see any now.